### DISCUSSION TITLE
Remember the Collapsed/Expanded state of created groups

### DISCUSSION THREAD

**Discussion by sayyid5416** [community/community#7932] [Projects and Issues]
Groups are a great way to organise the issues into certain types. We can collapse them if we don't want all the issues to be thrown at our faces altogether.

But **problem is they expand, everytime I refresh the webpage**. If somehow they could remember, which groups were expanded, and which ones were collapsed. It would be a great convenience.

**Comment by tkvangorder** [comment]
We agree with this, it would be really handy to remember the collapsed/expanded state when you are grouping issues.

**Comment by mikermcneil** [comment]
~~relates to https://github.com/github/feedback/discussions/10080#discussioncomment-2418903~~

actually nevermind, that's re: projects 2.0 but I think this issue is about something else
